core/src/actor.rs

```diff
@@ -30,37 +37,51 @@ pub enum Protocol {
     /// HashTable::modify()
     Modify {
         keys: Keys,
-        old_pair: Pair,
-        new_pair: Pair,
+        old_entry: Entry,
```

Comment:
doesn't meta _need_ to be for a pair? entries aren't even unique across types...

e.g. can't i retract system entries like genesis if i send the retract meta for any other entry with the same content as gensis?

Reply:
For that, my thought was that certain operations, like retract, are not aloud on system entry types.